### PR TITLE
Travis updates

### PR DESCRIPTION
**Pull request by pvenkatakrishnan**
- remove grunt
- add istanbul for code coverage
- plugged in linting and tape directly into npm command line

### CODE REVIEW

**Comment by pvenkatakrishnan** [comment]
For https://github.com/krakenjs/spud/issues/9

File: package.json [hunk 1]
```diff
@@ -29,10 +31,8 @@
   },
   "devDependencies": {
     "chai": "*",
```
Comment (totherik) [inline package.json]: I don't believe chai is still in use.

**Comment by pvenkatakrishnan** [comment]
removed chai

**Comment by totherik** [comment]
LGTM

**Comment by grawk** [comment]
:+1:

**Pull request merged**